dato/js/recview.js

```diff
@@ -0,0 +1,158 @@
+$(document).ready( function(event){
+  // Loads Predictive Service client
+  // once it loads it triggers the recommenderView function
+  function loadScript(url){
+
+      var script = document.createElement("script")
+      script.type = "text/javascript";
+
+      if (script.readyState){  //IE
+          script.onreadystatechange = function(){
+              if (script.readyState == "loaded" ||
+                      script.readyState == "complete"){
+                  script.onreadystatechange = null;
+                  callback();
+              }
+          };
+      } else {  //Others
+          script.onload = function(){
+              recommenderView();
+          };
+      }
+
+      script.src = url;
+      document.getElementsByTagName("head")[0].appendChild(script);
+  }
+
+  loadScript('http://static.dato.com/files/scripts/predictive-service-client/ps-client.js');
```

Comment:
Shouldn't this URL be versioned? What happens if the JS API changes, or the compatibility w.r.t. versions of PS changes? (Are we guaranteeing backward compatibility on both fronts when we ship new versions of this client?)